### ISSUE TITLE
Feature request: UI zur Config

### ISSUE THREAD

**Issue by krobipd** [holomekc/ioBroker.hueemu#4]
Serwus

Ich finde dein Modul äußerst interessant und soweit ich das verstehe könnte es ja die HA-Bridge, Vorhallen in Zusammenarbeit mit dem iobroker, ersetzen.

Habe ich das richtig verstanden?
Ich kann zwar selbst so configs schreiben, das wäre ja nicht das problem aber dadurch ist die einstiegshürde doch sehr hoch und schwer.

Wäre toll wenn du vl über eine UI nachdenken würdest vl ala yahka. Damit könnte man das dann wunderbar und auch benutzerfreundlich Konfigurieren und nutzen.

**Comment by holomekc** [comment]
Daran arbeite ich gerade. Im iobroker Forum gibt es ein paar Wireframes. Ist bei mir aktuell halt eine Zeit frage. Könnte ein bissle dauern.

**Comment by Apollon77** [comment]
Wie wäre es mit der Idee als Start:

erlaube ein device json zu erstellen und dann anzulegen. Dabei könnte man verschiedene templates zur Auswahl anbieten für verschiedene device Typen. Ein json Editor ist nicht soo schwierig im admin. 

Noch mehr fancy wäre wenn man für das device Schema ggf json Schema nutzen kann bzw eins definiert. So könnte man daraus sogar automatisch eine UI zum komfortablen Editoren bauen. Ggf könnte hier @UncleSamSwiss mit Infos und Beispielen zur Seite stehen.

**Comment by UncleSamSwiss** [comment]
Ja, wir haben bei ham ein GUI gebaut, das mit JSON Schema arbeitet, das könnte hier auch hilfreich sein. Wenn es ein Schema gibt, kann man sehr einfach damit ein komplexes Formular machen: https://github.com/ioBroker/ioBroker.ham/blob/master/admin/tab_m.html. Wenn ich unterstützen kann, dann bitte einfach melden (hier oder im Forum als UncleSam).
